src/query/src/dist_plan/merge_scan.rs

```diff
@@ -192,7 +195,12 @@ impl MergeScanExec {
                     header: Some(RegionRequestHeader {
                         tracing_context: tracing_context.to_w3c(),
                         dbname: dbname.clone(),
-                        timezone: tz.clone(),
+                        query_context: Some(QueryContext {
+                            current_catalog: current_catalog.clone(),
+                            current_schema: current_schema.clone(),
+                            timezone: timezone.clone(),
+                            extension: extensions.clone(),
```

Comment:
Could you please rename the proto message field to extensions?
https://github.com/GreptimeTeam/greptime-proto/pull/156/files#r1587153168

Reply:
done